tsfeatures/tests/test_mutability.py

```diff
@@ -1,31 +0,0 @@
-import numpy as np
```

Comment:
Please add these tests to nbs/tsfeatures.ipynb

Reply:
added the tests to the tsfeatures.ipynb